src/common/parsers/AbstractTimeExpressionParser.ts

```diff
@@ -3,23 +3,23 @@ import { ParsingComponents, ParsingResult } from "../../results";
 import { Meridiem } from "../../index";
 
 // prettier-ignore
-function primaryTimePattern(primaryPrefix: string, primarySuffix: string) {
+function primaryTimePattern(leftBoundary: string, primaryPrefix: string, primarySuffix: string, flags: string) {
     return new RegExp(
-        "(^|\\s|T|\\b)" +
+        leftBoundary +
```

Comment:
Could you follow the existing pattern and use the inline parameter format?

Reply:
Actually, if you look at more recent code, we have been using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) than normal string, could you also help change everything here into template literals?

Reply:
Can you please elaborate on this? 
You mean using template literals like here for all lines from 8 to 21?
https://github.com/wanasit/chrono/blob/4e02153671af4e2895154860a78126aa036f8566/src/locales/en/parsers/ENCasualYearMonthDayParser.ts#L14

Reply:
Yes. Let's use the template literals for every string that makes up any pattern. 

In addition to the consistency, it would make injecting/replacing with parameters more natural.